### PR TITLE
applications: nrf_desktop: Dongle on nrf52840 DK

### PR DESCRIPTION
**Pull request by pdunaj**
Add configuration files to run dongle role on DK.

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

File: applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf [hunk 1]
```diff
@@ -0,0 +1,170 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
```
Comment (MarekPieta) [inline applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf:2]: 2020

File: applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+CONFIG_DESKTOP_INIT_LOG_MOTION_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT=n
+
+CONFIG_DESKTOP_HID_REPORT_DESC="configuration/common/hid_report_desc.c"
+
+CONFIG_DESKTOP_BUTTONS_ENABLE=y
+CONFIG_DESKTOP_BUTTONS_POLARITY_INVERSED=y
+
+CONFIG_DESKTOP_HID_DONGLE=y
```
Comment (MarekPieta) [inline applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf:18]: Click detector is necessary to control Bluetooth peers (ble_bond reacts on click event).
We could even add this as dependency.

File: applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf (hunk 1 above)
Comment (MarekPieta) [reply applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf:2]: Also for other files

File: applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf (hunk 2 above)
Comment (pdunaj) [reply applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf:20]: Added

File: applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf (hunk 1 above)
Comment (pdunaj) [reply applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf]: Fixed

**Pull request merged**